module/executiondatasync/tracker/storage.go

```diff
@@ -0,0 +1,454 @@
+package tracker
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/dgraph-io/badger/v2"
+	"github.com/hashicorp/go-multierror"
+	"github.com/ipfs/go-cid"
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/module/blobs"
+)
+
+// badger key prefixes
+const (
+	prefixGlobalState  byte = iota + 1 // global state variables
+	prefixLatestHeight                 // tracks the latest height containing each blob
```

Comment:
Can we explicitly document which height we're referring to here and elsewhere in this file where we refer to height? Given an execution receipt like:
```
A<-B(Receipt_A)
```
are we referring to `A.height` - the height of the executed block - or `B.height` - the height of the container block.